### ISSUE TITLE
some cron strings take a long time to schedule

### ISSUE THREAD

**Issue by jmettraux** [jmettraux/rufus-scheduler#188]
As somehow reported in http://stackoverflow.com/questions/34446354 by @kssminus

https://gist.github.com/kssminus/8e26f1950056011a3499 (to be refined)

**Comment by jmettraux** [comment]
refined into https://gist.github.com/jmettraux/b53f2c1932d9bdcc489a

```ruby
["* * * * *", 0.093]
["*/5 * * * *", 12.525]
["*/10 * * * *", 10.447]
["10 4 * * *", 0.707]
["35 */3 * * *", 3.65]
["* */3 * * *", 0.017]
["16 */4 * * *", 2.744]
["30 14 * * *", 0.606]
["26 */2 * * *", 5.283]
```

Seems like `*/5 * * * *` and `*/10 * * * *` are the costliest.

**Comment by jmettraux** [comment]
Caused by #89

**Comment by jmettraux** [comment]
Please double-check.

**Comment by jmettraux** [comment]
@kssminus please double-check this solution.

**Comment by kssminus** [comment]
@jmettraux awesome! It's six times faster than before. It's too late but, Merry Chrismas. :santa:

**Comment by jmettraux** [comment]
Ah, that's good news. Merry Late Christmas to you as well.

I'll package that into 3.2.0 and release.

새해 복 많이 받으세요!

**Comment by jmettraux** [comment]
https://dl.dropboxusercontent.com/u/76595/blog/20151228.html released rufus-scheduler 3.2.0
